app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -57,20 +76,11 @@ def before_oplog_inserts(&block)
         def before_building_model(&block)
           before_building_model_callbacks << block
         end
-      end
 
-      NATIONAL = 'N'.freeze
-      APPROVED_FLAG = '1'.freeze
-      STOPPED_FLAG = '1'.freeze
-      PATH_SEPARATOR = '.'.freeze
-      METAINFO = 'metainfo'.freeze
-      BASE_MAPPING = {
-        'validityStartDate' => :validity_start_date,
-        'validityEndDate' => :validity_end_date,
-        'metainfo.origin' => :national,
-        'metainfo.opType' => :operation,
-        'metainfo.transactionDate' => :operation_date,
-      }.freeze
+        def primary?
+          name.match(/\ACdsImporter::EntityMapper::(?<entity_class>.*)Mapper\z/).try(:[], :entity_class) == entity_class
+        end
```

Comment:
this is pretty dense,
you can improve readability here, maybe splitting it into two lines?

```
entity_mapper = name.match(/\ACdsImporter::EntityMapper::(?<entity_class>.*)Mapper\z/)

entity_mapper.try(:[], :entity_class) == entity_class
```

Reply:
it is not clear how the concept of `primary` is deduced?